src/main/java/io/reactivex/Observable.java

```diff
@@ -11707,12 +11707,18 @@ public final void subscribe(Observer<? super T> observer) {
         Flowable<T> o = new FlowableFromObservable<T>(this);
 
         switch (strategy) {
-        case DROP:
-            return o.onBackpressureDrop();
-        case LATEST:
-            return o.onBackpressureLatest();
-        default:
-            return o.onBackpressureBuffer();
+            case DROP:
+                return o.onBackpressureDrop();
+            case LATEST:
+                return o.onBackpressureLatest();
+            case NONE:
+                return o;
+            case ERROR:
+                return o.onBackpressureError();
+            case BUFFER:
```

Comment:
Please leave these as `default:` because otherwise the default case can't be coverage-tested.